packages/cosmic-proto/proto/cosmos/crypto/hd/v1/hd.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
```

Comment:
It's missing the [first line](https://github.com/cosmos/cosmos-sdk/blob/7799bba7bc889288607576aa11655b5fc31a2da9/proto/cosmos/crypto/hd/v1/hd.proto#L1) of the version from cosmos-sdk 0.46.16.

A trivial difference to be sure, but it just makes me uncertain whether these protos were copied from the right place. I'll await the previously-requested info about where these came from.